src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,36 @@
+package mate.academy.springboot.datajpa.service.impl;
+
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.repository.CategoryRepository;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.stereotype.Service;
+
+@Service

```

Comment:
Better make your solution consistent. if you use Lombok annotations instead constructor initialization use it everywhere.